app/_includes/plugins/hub-examples/consumer_groups.html

```diff
@@ -0,0 +1,74 @@
+{%- assign hub_example = include.hub_example -%}
+{%- assign kong_manager_fields = hub_example.kong_manager.fields -%}
+
+{% if include.render_intro_text %}
+The following examples provide some typical configurations for enabling
+the <code>{{ hub_example.plugin_name }}</code> plugin on a
+[consume group](gateway/latest/admin-api/consumer-groups/reference/).
+{% endif %}
+
+{% navtabs %}
+  {% if hub_example.render_curl? %}
+  {% navtab Admin API %}
+
+Make the following request:
+
+{% include_cached plugins/hub-examples/curl.html example=hub_example.curl %}
+
+Replace `CONSUMER_GROUP_NAME|CONSUMER_GROUP_ID` with the `id` or `name` of the consumer that this plugin configuration will target.
+
+{% if page.service_id and page.route_id %}
+You can combine `consumer.id`, `service.id`, or `route.id`
+{% elsif page.service_id %} and `service.id`
+{% elsif page.route_id %} and `route.id`
+{% endif %} in the same request, to further narrow the scope of the plugin.
+
+  {% endnavtab %}
+  {% endif %}
+
+  {% unless page.k8s_examples == false %}
+    {% if hub_example.render_kubernetes? %}
+    {% navtab Kubernetes %}
+
+First, create a [KongPlugin](/kubernetes-ingress-controller/latest/references/custom-resources/#kongplugin)
+resource:
+
+{% include_cached plugins/hub-examples/kubernetes.html example=hub_example.kubernetes %}
+
+Next, apply the `KongPlugin` resource to an ingress by annotating the `KongConsumerGroup` object as follows:
+
+```yaml
+kubectl annotate KongConsumer CONSUMER_NAME konghq.com/plugins={{ hub_example.kubernetes.plugin_name }}-example
+```
+
+Replace `CONSUMER_GROUP_NAME` with the name of the consumer that this plugin configuration will target.
+You can see your available consumers by running `kubectl get KongConsumer`.
+
+To learn more about `KongConsumer` objects, see [Provisioning Consumers and Credentials
+](/kubernetes-ingress-controller/latest/guides/using-consumer-credential-resource/).
+
+<blockquote class="note">
+  <strong>Note:</strong> The KongPlugin resource only needs to be defined once
+  and can be applied to any service, consumer group, or route in the namespace. If you
+  want the plugin to be available cluster-wide, create the resource as a
+  <code>KongClusterPlugin</code> instead of <code>KongPlugin</code>.
+</blockquote>
```

Comment:
```suggestion
<blockquote class="note">
  <strong>Note:</strong> The KongPlugin resource only needs to be defined once
  and can be applied to any service, consumer, consumer group, or route in the namespace. If you
  want the plugin to be available cluster-wide, create the resource as a
  <code>KongClusterPlugin</code> instead of <code>KongPlugin</code>.
</blockquote>
```